sdks/java/harness/src/main/java/org/apache/beam/fn/harness/state/FnApiStateAccessor.java

```diff
@@ -576,6 +577,17 @@ public ReadableState<Boolean> readLater() {
             });
   }
 
+  @Override
+  public <KeyT, ValueT> MultimapState<KeyT, ValueT> bindMultimap(
+      String id,
+      StateSpec<MultimapState<KeyT, ValueT>> spec,
+      Coder<KeyT> keyCoder,
+      Coder<ValueT> valueCoder) {
+    throw new UnsupportedOperationException(
+        "TODO(https://github.com/apache/beam/issues/23616): Add support for a multimap"
```

Comment:
Maybe just move the TODO as a comment and the exception just says multimap not supported.

Reply:
Done